modules/nw-http2-haproxy.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-http2-haproxy{context}"]
+= HTTP/2 Ingress connectivity
+
+You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. It allows application owners to make use of HTTP/2 protocol capabilities, including single connection, header compression, binary streams, and more.
```

Comment:
Is this too much conceptual exposition prior to a procedure? Right now there's probably nowhere else for it to land.

Reply:
Probably, it doesn't need to stand alone so I think it's okay for now. In later iterations (when we reorg networking content) this could become a conceptual module.